modules/ucfunnelBidAdapter.js

```diff
@@ -210,12 +211,39 @@ function getSupplyChain(schain) {
   return supplyChain;
 }
 
+function getMediaType(mediaTypes) {
+  if (mediaTypes != null && mediaTypes.video) {
+    return 'video';
+  } else if (mediaTypes != null && mediaTypes.native) {
+    return 'native'
+  }
+  return 'banner';
+}
+
+function getFloor(bid, size, mediaTypes) {
+  if (bid.params.bidfloor) {
+    return bid.params.bidfloor;
+  }
+  if (typeof bid.getFloor === 'function') {
+    var bidFloor = bid.getFloor({
+      currency: CURRENCY,
+      mediaType: getMediaType(mediaTypes),
```

Comment:
On multiformat unit,  this appears to slays at the video floor

Reply:
Does multiformat unit mean that adUnits support multiple mediaType ? 
Like support banner and video in the same adUnits.

Reply:
I update the commit to fix the banner bid floor problem.

Reply:
@patmmccann any comment about the update commit?